packages/gestalt/src/Masonry/defaultLayout.ts

```diff
@@ -100,7 +100,7 @@ const defaultLayout =
         })
       : items.reduce<Array<any>>((acc, item) => {
           const positions = acc;
-          const height = measurementCache.get(item);
+          const height = measurementCache.get(item) || (item as any).height;
```

Comment:
this is needed or else the loading page is empty -- wondering if we should update the type to make `height` required?

eg: `U extends { height: number }` (of course, this would mean updating a bunchhhh of types 😅 )

Reply:
So for the skeleton items we don't render them first to get the height? This is something that could be good to look into more detail.

Reply:
lets discuss at the next Masonry sync 👍

Reply:
closing the loop here from our discussion:

- we should not render the items first since we will always know what the height is
  - make height required